doc/quick-install.md

```diff
@@ -25,6 +25,14 @@ Finally on the same machine test that you can ping the FQDN with: `ping "$(hostn
 echo 'deb https://download.jitsi.org stable/' >> /etc/apt/sources.list.d/jitsi-stable.list
 wget -qO -  https://download.jitsi.org/jitsi-key.gpg.key | sudo apt-key add -
 ```
+### Open ports in your firewall
+
+Open the following ports in your firewall, to allow traffic to the machine running jitsi:
+
+ - 80 TCP
```

Comment:
@neilzone @damencho is it a great idea to recommend opening TCP port 80 as part of the quck-install guide?

It seems like that could lead some users to unintentionally expose plaintext traffic to-and-from their Jitsi instance.

Reply:
I’m not sure you’d be able to get a Let’s Encrypt certificate without it?

Reply:
Thanks - yep, after reading up on [certbot](https://certbot.eff.org/), you're correct (at least for automatic certificate provisioning - and it's probably reasonable to assume that's what people will use for a quick install).

Your suggested clarifications in https://github.com/jitsi/jitsi-meet/pull/5940/commits/6cf8e179d9c8d95a5c6b78dad50841e2efcda622#diff-39d47d1432f7af0aa02f3e913097320aR43 look good to me too.